xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -345,6 +345,47 @@ impl<Network: Get<Option<NetworkId>>, AccountId: From<[u8; 20]> + Into<[u8; 20]>
 	}
 }
 
+/// Converts a location which is a top-level relay chain (which provides its own consensus) into a 32-byte `AccountId`.
+///
+/// This will always result in the *same account ID* being returned for the same Relay-chain, regardless of the relative security of
+/// this Relay-chain compared to the local chain.
+///
+/// Note: No distinction is made when the local chain happens to be the relay chain in
+/// question or its Relay-chain.
+///
+/// WARNING: This results in the same `AccountId` value being generated regardless
+/// of the relative security of the local chain and the Relay-chain of the input
+/// location. This may not have any immediate security risks, however since it creates
+/// commonalities between chains with different security characteristics, it could
+/// possibly form part of a more sophisticated attack scenario.
+pub struct GlobalConsensusConvertsFor<UniversalLocation, AccountId>(
```

Comment:
This converter doesn't work with **this** network location - i.e. if I'll call it with `MultiLocation::new(2, Here)`, it'll return error. So maybe change the name to something like `RemoteGlobalConsensusConvertsFor`? Current one is ok too, but maybe worth adding some comments about this behavior to the struct documentation?

Reply:
(and what I mean under "conflict" is that without using `Remote` or `Foreign` I may imagine that what goes first, gets a priority and otherwise I'll know for sure that this `GlobalConsensusConvertsFor` won't trigger for the parent/relay chain)